database/Table_Insert.py

```diff
@@ -20,7 +20,7 @@ def insert_data(name, data_list, session_value, has_id=False):
 
     if session_value is not None:
         session_recorded_sessions = cursor.execute('select count(session) from sessions where session = "' + session_value + '" and table_name = "' + name + '"').fetchall()
-        if session_recorded_sessions[0][0] > 0:
+        if session_recorded_sessions[0][0] > 0 or name == 'p0f':
```

Comment:
I would think you would want to check "name == 'p0f' " up in the "if session_value is not None and name is not 'p0f'  otherwise you are doing select without needing to.